nipoppy_cli/docs/source/conf.py

```diff
@@ -14,6 +16,16 @@
 copyright = "2024, NeuroDataScience-ORIGAMI Lab"
 author = "NeuroDataScience-ORIGAMI Lab"
 
+# The version info for the project you're documenting, acts as replacement
+# for |version| and |release|, also used in various other places throughout
+# the built documents.
+#
+# The short X.Y version.
+version = __version__
+
+# The full version, including alpha/beta/rc tags.
+release = __version__
```

Comment:
I'm not sure to understand how the `version` and `release` will be different since they both refer to the `__version__` value.

Reply:
I think that this is more to provide a value for any HTML template that may need it.

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-release